File: 프로그래머스/0/181949. 대소문자 바꿔서 출력하기/대소문자 바꿔서 출력하기.cs
using System;

public class Example
{
    public static void Main()
    {
        String s;
        String a;
        String str = "";
        
        Console.Clear();
        s = Console.ReadLine();
        
        for (int index = 0; index < s.Length; index++){
            a = s.Substring(index, 1);
            if(char.IsUpper(a[0])) {
                str += char.ToLower(a[0]);
            }
            else if(char.IsLower(a[0])) {
                str += char.ToUpper(a[0]);
            }
            
        }
        Console.WriteLine(str);
    }
}

File: 프로그래머스/0/181949. 대소문자 바꿔서 출력하기/대소문자 바꿔서 출력하기.cpp
#include <iostream>
#include <string>

using namespace std;

int main(void) 
{
    string str;
    cin >> str;
    
    for (char &c : str) // 모든 문자를 순회
    {
        if (isupper(c)) // 문자가 대문자인 경우
        {
            c = tolower(c); // 소문자로 변환
        }
        else if (islower(c)) // 문자가 소문자인 경우
        {
            c = toupper(c); // 대문자로 변환
        }
    }
    cout << str;
    return 0;
}